lib/mailserver.js

```diff
@@ -25,7 +25,6 @@ var config = {};
 var store = [];
 var tempDir = path.join(os.tmpdir(), 'maildev', process.pid.toString());
 var eventEmitter = new events.EventEmitter();
-var smtp;
```

Comment:
This is now a local variable, which I unfortunately forgot to define. Will do so.